docs/guides/all-keys-one-secret.md

```diff
@@ -18,10 +18,53 @@ Now, when creating our ExternalSecret resource, instead of using the data field,
 ```yaml
 {% include 'gcpsm-data-from-external-secret.yaml' %}
 ```
+Here, "example" is the name of the external secret that will be created in our cluster.    
+Whereas, "secret-to-be-created" is the name of Kubernetes secrets that will be created.    
+Note: Since these secrets are namespace-based resources, so you can also explicitly specify the "namespace" under the "metadata" block of the above external secret file.    
+when we use, 
 
-To check both values we can run:
+```
+  dataFrom:
+  - extract:
+      key: all-keys-example-secret
+```
+We get all the key-value pairs present over the remote secret store (GCP or AWS or Azure) and can pass either all or a few key-values as environment variables.    
+Please note that, "all-keys-example-secret" is the name of your secret present on GCP/AWS secrets manager/Azure    
+    
+We can pass a few secrets as env variables as below:
+```
+        env:
+          - name: key1
+            valueFrom:
+              secretKeyRef:
+                name: secret-to-be-created
+                key: username
 
+          - name: key2
+            valueFrom:
+              secretKeyRef:
+                name: secret-to-be-created
+                key: surname
+```
+ 
+Here,    
+\<key1\> and \<key2> are the names of keys that will be created and passed as env variables.    
+\<secret-to-be-created\>: is the name of your Kubernetes secret created by you.    
+\<username\> and \<password>: is the particular key in the secrets manager whose value you want to pass.    
+To check both values we can run:
 ```
 kubectl get secret secret-to-be-created -n <namespace> -o jsonpath='{.data.username}' | base64 -d
 kubectl get secret secret-to-be-created -n <namespace> -o jsonpath='{.data.surname}' | base64 -d
```

Comment:
```suggestion
kubectl get secret secret-to-be-created -n <namespace> -o jsonpath='{.data.password}' | base64 -d
```

Reply:
Resolved it. I have updated the <username\> and \<password> to <username\> and \<surname> as "surname" was being used in the above image as well as in the command "kubectl get secret secret-to-be-created -n <namespace> -o jsonpath='{.data.surname}' | base64 -d".